flask_resources/deserializers/deserializers.py

```diff
@@ -5,12 +5,12 @@
 # Flask-Resources is free software; you can redistribute it and/or modify it
 # under the terms of the MIT License; see LICENSE file for more details.
 
-"""Loaders required interfaces."""
+"""Deserializers required interfaces."""
 
 
-class LoaderMixin:
-    """Loader Interface."""
+class DeserializerMixin:
+    """Deserializer Interface."""
 
-    def load_request(self, *args, **kwargs):
-        """Load the body of the request."""
+    def deserialize_object(self, obj, *args, **kwargs):
+        """Deserializes an object."""
         raise NotImplementedError()
```

Comment:
Futuristically speaking, in the same way that we serialize an object list, for bulk operations we will need to deserialize lists... So this interface will have that extra method (not implemented yet since is not used in any default, but will make it easier to have a class).

About `obj` naming... You are correct, I'm changing it for "data", for now... Let's review naming later on when we actually implement something on top and things are more clear?